src/aks-preview/azext_aks_preview/_client_factory.py

```diff
@@ -17,6 +17,8 @@
 def get_container_service_client(cli_ctx, **_):
     return get_mgmt_service_client(cli_ctx, CUSTOM_MGMT_AKS_PREVIEW)
 
+def cf_trustedaccess(cli_ctx, *_):
+    return get_container_service_client(cli_ctx).trusted_access_roles
```

Comment:
1. trusted_access_roles-> trustedaccess_roles
2. what is goal of `cf_trustedaccess`, if it is to support both role & binding, we cannot simply return roles here?

Reply:
ok